cypress/integration/ui/create_dashboard/index.js

```diff
@@ -0,0 +1,111 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps'
+
+const UID_LENGTH = 11
+
+const dashboardTitle = new Date().toString()
```

Comment:
Consider changing this to something like `TEST_DASHBOARD_TITLE` to indicate that it's a test string?

Reply:
Done